### ISSUE TITLE
Hedgehog seems to builds from source

### ISSUE THREAD

**Issue by codygman** [utdemir/hs-nix-template#12]
I'm stuck on windows for a bit so I'm trying out hs-nix-template from a new VM. It seems that hedgehog has to be built from source for some reason and that takes forever and it seems its dependencies also have to be built from source.

```
[demo@nixos:~/code/test-nix-template]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.4.35, NixOS, 20.03.1577.74a80c5a9ab (Markhor)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.4`
 - channels(root): `"nixos-20.03.1577.74a80c5a9ab"`
 - channels(demo): `"home-manager"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by utdemir** [comment]
Hi. That's interesting. I just checked and it is indeed not present in `cache.nixos.org`. It looks like we almost get all of our packages from the cache if we use `nixpkgs-unstable` branch. I made that change alongside with a GHC version bump on this PR: https://github.com/utdemir/hs-nix-template/pull/13

Also, this project has a cachix cache at https://app.cachix.org/cache/hs-nix-template , which is automatically populated by CI. Ideally it shouldn't be necessary since we want `hs-nix-template` to require no system configuration changes, but it might be helpful as an interim solution.
